### PR TITLE
add Fitbit source

### PR DESCRIPTION
**Pull request by beaugunderson**

### CODE REVIEW

**Comment by beaugunderson** [comment]
@madprime ready for review :)

File: sources/fitbit.py [hunk 1]
```diff
@@ -0,0 +1,379 @@
...
+            'tags': ['weight', 'Fitbit', 'steps', 'activity'],
+        }
+    }]
+
+    fitbit_data = get_fitbit_data(access_token)
+
+    if fitbit_data['rate_cap_encountered']:
+        # If this is previously called and we got no new data this round,
+        # double the wait period for resubmission.
+        if 'return_status' in kwargs and not fitbit_data['all_data']:
```
Comment (madprime) [inline sources/fitbit.py:309]: Confession: I think this is copied from my approach in Moves, and I'm pretty sure it isn't working like it should. Cached data is going to result in 'all_data' being filled, even without any successful API calls. (I don't think this is a blocker.)

**Comment by madprime** [comment]
Looks good to me, :+1: to merge into staging and test there!

File: sources/fitbit.py (hunk 1 above)
Comment (beaugunderson) [inline sources/fitbit.py:309]: `all_data` is filled on each pass through (constructed by calling `fitbit_query`, which gathers the data from either Fitbit or the cache), so `all_data` itself is never cached. :)

**Pull request merged**